### ISSUE TITLE
Typing in comments scrolls page

### ISSUE THREAD

**Issue by nellh** [OpenNeuroOrg/openneuro#1712]
**Describe the bug**
The page scrolls out of view if you are typing in a comment field below the fold.

**To Reproduce**
1. Visit a dataset with a lot of files.
2. Start typing in comments.
3. Page will scroll comment input out of view.

**Expected behavior**
Page should not randomly scroll.

**Screenshots**

**Desktop (please complete the following information):**
- OS: Linux
 - Browser: Firefox 77 and Chrome 83

**Smartphone (please complete the following information):**
 - OS: Android 10
 - Browser: Firefox Preview

**Comment by nellh** [comment]
Also seeing this on the dashboard on mobile but it seems less of an issue there. Could be the same CSS causing the issue?

**Comment by nellh** [comment]
This looks like it could be cause by an extra render triggered by the error boundary component. Chrome preserves the position most of the time but Firefox it is easily reproducible if the page is scrollable.

**Comment by thinknoack** [comment]
please remove

```
.dataset-route{
    -webkit-transform-style: preserve-3d;
    transform-style: preserve-3d;
} 
```

**Comment by nellh** [comment]
Fix released in 3.18.0
